test/integration/targets/ansible-galaxy/runme.sh

```diff
@@ -2,6 +2,14 @@
 
 set -eux -o pipefail
 
+# Prep the local git repos with role and make a tar archive so we can test
+# different things
+galaxy_local_test_role="test-role"
+galaxy_local_test_role_dir=$(mktemp -d)
+galaxy_local_test_role_git_repo="${galaxy_local_test_role_dir}/${galaxy_local_test_role}"
+galaxy_local_test_role_tar="${galaxy_local_test_role_dir}/${galaxy_local_test_role}.tar"
+galaxy_webserver_root="/tmp/ansible-galaxy-webserver"
```

Comment:
Is there a reason a directory under `$OUTPUT_DIR` cannot be used instead of `/tmp/`?

Reply:
The reason is I didn't know it was available :-D Fixed in latest revision.